net/jool/Makefile

```diff
@@ -8,14 +8,14 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=jool
 PKG_VERSION:=3.5.2
-PKG_RELEASE:=1
+PKG_RELEASE:=2
 
 PKG_LICENSE:=GPL-3.0
 PKG_LICENSE_FILES:=COPYING
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://github.com/NICMx/Jool.git
-PKG_SOURCE_VERSION:=ec32f7181d6b12eadcafbe39b8bf4d7d48230232
+PKG_SOURCE_VERSION:=66a791c90751d7941b08c142babe1deec73d0996
```

Comment:
Nice to know this commit fixes an issue, but it not version `3.5.2` anymore.  The other options could be

 - Change `PKG_VERSION` to a date versioned one like other packages
 - Incorporate the change as a new patch